packages/cozy-device-helper/src/link.js

```diff
@@ -0,0 +1,26 @@
+import { hasSafariPlugin, hasInAppBrowserPlugin } from './plugins'
+
+export const nativeLinkOpen = async ({ url }) => {
+  if ((await hasSafariPlugin()) && window.SafariViewController) {
```

Comment:
[`hasSafariPlugin` fait déjà un check sur `window.SafariViewController`](https://github.com/cozy/cozy-libs/blob/9244f76ad0e9a9540a1fc121ded4cf8616392bdd/packages/cozy-device-helper/src/plugins.js#L11)

```suggestion
  if (await hasSafariPlugin()) {
```